### DISCUSSION TITLE
why progress bar for wire:navigate just show once

### DISCUSSION THREAD

**Discussion by restu-lomboe** [livewire/livewire#7727] [1. Help]
PHP v8.1
Laravel v10.10
livewire v3.10

why if I use wire:navigate, the loading progress bar will only appear once, on the next page when we click wire:navigate the loading progress bar does not appear, but if we refresh first then click wire:navigate the loading progress bar appears again

**Comment by joshhanley** [comment]
We're in the process of investigating all wire:navigate issues and we can't replicate this with the information give so I'm going to close this. If you're still experiencing problems, please open a new discussion and provide code examples that demonstrate the issue that we can copy and paste so we can investigate. Thanks!
